.github/workflows/release-package.yml

```diff
@@ -0,0 +1,28 @@
+name: Release Package
+
+on:
+  push:
+    branches: [ $default-branch ]
+    types: [closed]
```

Comment:
What does type closed mean?  Pushes to the develop branch shouldn't have an open/closed status (we're triggering on the push itself not the PR)